test/MeasureService.test.ts

```diff
@@ -0,0 +1,42 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
+import { serverConfig } from '../src/config/serverConfig';
+import { cleanUpDb, testSetup } from './utils';
+const supertest = require('supertest');
+
+let server: Server;
+
+const Measure: fhir4.Measure = { resourceType: 'Measure', id: 'measure123', status: 'active' };
+
+describe('measure.service', () => {
```

Comment:
Same comments about the test styling here. Also, the describe is named `measure.service` here but `LibraryService` in the library service. Can we be consistent?